code/src/components/NewTodo.js

```diff
@@ -0,0 +1,78 @@
+import React, { useState } from 'react'
+import { useDispatch } from 'react-redux'
+import styled from 'styled-components'
+import uniqid from 'uniqid'
+
+import todos from '../reducers/todos'
+
+
+const Input = styled.input`
+  width:100%;
+  border: none;
+  outline:none;
+  background: none;
+  font-size: 18px;
+  color: #555;
+  padding:20px 10px 20px 5px;
+  margin: 30px 0 15px 0;
+  border-radius: 25px;
+  box-shadow: inset 8px 8px 8px #cbced1,
+              inset -8px -8px 8px #ffffff;
```

Comment:
In line 15 you used a nice shorthand version of hex-code, you can use it here as well 🏃🏻